May update first draft (#414)
simplabs/simplabs.github.io: blog/_posts/2019-05-08-may-monthly-update.md

---
layout: article
section: Blog
title: "simplabs monthly update - May 2019"
author: "Mark Coleman"
github-handle: mrmrcoleman
twitter-handle: mrmrcoleman
topic: simplabs
---

Welcome to the third installment of our monthly update. Each month we cover the events and activities that have been happening at simplabs along with the things we're looking forward to. Enjoy.

<!--break-->

## News

#### Samantha de Barros joins simplabs

> We’re very excited to announce that [Samantha de Barros](https://twitter.com/sami_dbc) has joined simplabs.

Reviewer: I think it'd be

```suggestion
We’re very excited to announce that [Samanta de Barros](https://twitter.com/sami_dbc) has joined simplabs.
```

from [her website](https://sdebarros.io/)

Reviewer 2: Agreed, and OMG how did you find that? 😄

Reviewer: Lol, always googling :-)

> Samantha started out her career as a .Net and Java developer before shifting her attentions to Ember and Rails as she preferred the agility of web applications. Her work as a consultant helped Samantha to build experience with many different types of companies. She has shared this experience in talks at events including EmberConf and JSDay Uy.

Reviewer: I think it'd be

```suggestion
Samanta started out her career as a .Net and Java developer before shifting her attentions to Ember and Rails as she preferred the agility of web applications. Her work as a consultant helped Samanta to build experience with many different types of companies. She has shared this experience in talks at events including EmberConf and JSDay Uy.
```

> Samantha started out her career as a .Net and Java developer before shifting her attentions to Ember and Rails as she preferred the agility of web applications. Her work as a consultant helped Samantha to build experience with many different types of companies. She has shared this experience in talks at events including EmberConf and JSDay Uy.

Author: @Turbo87 This one? https://www.youtube.com/watch?v=31kVznd-zys

> Samantha started out her career as a .Net and Java developer before shifting her attentions to Ember and Rails as she preferred the agility of web applications. Her work as a consultant helped Samantha to build experience with many different types of companies. She has shared this experience in talks at events including EmberConf and JSDay Uy.

Author: Done

> Our CEO Marco says _“I'm very happy to have Samanta on board. Her expertise with both Ember.js and Rails as well as experience with consulting international clients and as a public speaker make her a great fit to our team and we all look forward to working with her.”_

Author: I think it still works but happy to change it if you have a better suggestion?

#### ember-intl-analyzer released

Yet another cool tool to emerge from out work with [Qonto](https://qonto.eu/en)!

The ember-intl-analyzer CLI tool analyzes your app code and finds unused translations for you and will soon provide warnings for missing and broken translations too.

Check it out here: [https://github.com/simplabs/ember-intl-analyzer](https://github.com/simplabs/ember-intl-analyzer)

#### simplabs is a founding sponsor of the Erlang Ecosystem Foundation

We are very pleased to be a founding sponsor of the Erlang Ecosystem Foundation alongside companies like Cisco, Whatsapp Inc. and Erlang Solutions.

The Erlang Ecosystem Foundation is a new non-profit organization dedicated to furthering the state of the art for Erlang, Elixir, LFE, and other technologies based on the BEAM.

Join the mailing list [here](https://erlef.org/) or [follow them on Twitter](https://twitter.com/TheErlef) to be kept up-to-date.

#### Other news

In other news [Tobi](https://twitter.com/TobiasBieniek) released [a well received blog post](https://simplabs.com/blog/2019/04/24/dependency-updates-for-gitlab.html) about automatically updating dependencies on internal Gitlab servers.

> [Chris Manson](https://twitter.com/real_ate) joined [Jan Weber](https://twitter.com/jwwweber/) for an episode of "May I ask a question", the weekly Emberjs Q&A livestream [here](https://www.youtube.com/watch?v=v1rBL5_KPqU).

Reviewer: I think it'd be

```suggestion
[Chris Manson](https://twitter.com/real_ate) joined [Jen Weber](https://twitter.com/jwwweber/) for an episode of "May I ask a question", the weekly Emberjs Q&A livestream [here](https://www.youtube.com/watch?v=v1rBL5_KPqU).
```

> And last but not least [Ricardo Mendes](https://twitter.com/locks) released a new episode of the Conversas em Código podcast, this time talking about [Twilio, WebSockets, Pheonix Presence and more](https://trello.com/c/8W25cdsV/28-episode-19-of-locks-portuguese-podcast).

Reviewer: I think it'd be

```suggestion
And last but not least [Ricardo Mendes](https://twitter.com/locks) released a new episode of the Conversas em Código podcast, this time talking about [Twilio, WebSockets, Phoenix Presence and more](https://trello.com/c/8W25cdsV/28-episode-19-of-locks-portuguese-podcast).
```

## Upcoming Events

_We're always interested in new speakers for the various meetups we're involved in. If you'd like to give a talk, [please get in touch.](https://simplabs.com/contact/index.html)_

#### Ember.js Berlin - Javascript for Beginners

On May 24th and 25th [Jessica Jordan](https://twitter.com/jjordan_dev) is leading a Javascript for beginners workshop.

> Day 1 is an installation party to make sure is that everybody is set up for day 2 when attendees will be guided through creating their first web application using EmberJS. The workshop is open to women and those who are non-binary, and you can sign up using the links below!

Reviewer: I think it'd be

```suggestion
Day 1 is an installation party to make sure that everybody is set up for day 2 when attendees will be guided through creating their first web application using EmberJS. The workshop is open to women and those who are non-binary, and you can sign up using the links below!
```

[Day 1](https://www.meetup.com/Ember-js-Berlin/events/260668921/) & [Day 2.](https://www.meetup.com/Ember-js-Berlin/events/260668987/)

#### Jessica Jordan at JSConfEU

> On the 1st of June [Jessica Jordan](https://twitter.com/jjordan_dev) is speaking at [JSConfEU](https://2019.jsconf.eu/) with her talk "Crafting Comics for Literally Everyone: How you can craft an immersive webcomic experience that accessible for blind and visually-impaired readers!"

Reviewer: I think it'd be

```suggestion
On the 1st of June [Jessica Jordan](https://twitter.com/jjordan_dev) is speaking at [JSConfEU](https://2019.jsconf.eu/) with her talk "Crafting Comics for Literally Everyone" she highlights how you can craft an immersive webcomic experience that is accessible for blind, visually-impaired and sighted readers alike!
```

Full talk details [here](https://2019.jsconf.eu/jessica-jordan/crafting-comics-for-literally-everyone.html).

#### Ricardo Mendes talking at Commit Porto

On the 22nd of June [Ricardo Mendes](https://twitter.com/locks) is giving the talk "Thriving through the hype cycle: an Ember.js story" at [CommitPorto](https://commitporto.com/).

Full talk details to follow soon.

#### Jessica speaking at FullStackCon London

In London this July [Jessica Jordan](https://twitter.com/jjordan_dev) is speaking at [FullStackConf](https://skillsmatter.com/conferences/11213-fullstack-london-2019-the-conference-on-javascript-node-and-internet-of-things) with her talk "Crafting Comics or Literally Everyone".

Follow [their Twitter](https://twitter.com/fullstackcon) for updates as the event approaches.

> #### EmberFestEU - October 17th & 18th - Earlybird tickets sold out

Reviewer 3: ```suggestion
#### EmberFest - October 17th & 18th - Earlybird tickets sold out
```

> ![EmberFestEU](/images/posts/2019-04-05-april-monthly-update/ember-fest-logo.png)

Reviewer 3: ```suggestion
![EmberFest](/images/posts/2019-04-05-april-monthly-update/ember-fest-logo.png)
```

> The [EmberFestEU](https://emberfest.eu/) early bird tickets are now sold out!

Reviewer 3: ```suggestion
The [EmberFest](https://emberfest.eu/) early bird tickets are now sold out!
```

> Taking place in Prague in October, EmberFestEU brings together the European Ember community for two days of talks and activities.

Reviewer: I think it'd be

```suggestion
Taking place in Copenhagen in October, EmberFestEU brings together the European Ember community for two days of talks and activities.
```

> Taking place in Prague in October, EmberFestEU brings together the European Ember community for two days of talks and activities.

Reviewer 3: ```suggestion
Taking place in Prague in October, EmberFest brings together the European Ember community for two days of talks and activities.
```

Grab a ticket [here](https://emberfest.eu/) while you still can and keep an eye on the [twitter](https://twitter.com/EmberFest) for updates as the conference takes shape.

## Past Events

#### Open Source Contribution workshop with Qonto at EmberJSParis

> On April 23rd [Tobi](https://twitter.com/TobiasBieniek/) and [Ricardo](https://twitter.com/locks) led an [open source contribution workshop](https://www.meetup.com/Paris-EmberJS-Lab/events/260514153/) with the EmberJSParis meetup group at the Qonto offices.

Reviewer 3: ```suggestion
On April 23rd [Tobias Bieniek](https://twitter.com/TobiasBieniek/) and [Ricardo Mendes](https://twitter.com/locks) led an [open source contribution workshop](https://www.meetup.com/Paris-EmberJS-Lab/events/260514153/) with the EmberJSParis meetup group at the Qonto offices.
```

The evening was a great success with many pull requests made to guides, addons and ember-cli... someone even wrote an RFC!

The event was later covered in [The Ember Times](https://the-emberjs-times.ongoodbits.com/2019/05/03/issue-96). Well done to all involved!

#### simplabs sponsored ElixirConf EU 2019

> We were proud sponsors on ElixirConfEU which took place in Prague on the 8th of April and by all accounts was a fantastic event.

Reviewer 3: ```suggestion
We were proud sponsors on ElixirConfEU which took place in Prague on the 8th and 9th of April and by all accounts was a fantastic event.
```

> #### Tobi's talk from EmberCamp Chicago 2019 released

Reviewer 3: ```suggestion
#### Tobias Bieniek's talk from EmberCamp Chicago 2019 released
```

> And lastly, looking way back to September 2018, [Tobi's](https://twitter.com/TobiasBieniek/) talk from [EmberCamp Chicago](https://twitter.com/embercamp), "Internationalization: It's Easy in Ember" was released and is [now available on YouTube.](https://www.youtube.com/watch?v=K4nKWp1z4cU&t=0s&list=PL4eq2DPpyBbm-vTgHMdBjUi1Qd5GiRIfW&index=5)

Reviewer 3: ```suggestion
And lastly, looking way back to September 2018, [Tobias Bieniek's](https://twitter.com/TobiasBieniek/) talk from [EmberCamp Chicago](https://twitter.com/embercamp), "Internationalization: It's Easy in Ember" was released and is [now available on YouTube.](https://www.youtube.com/watch?v=K4nKWp1z4cU&t=0s&list=PL4eq2DPpyBbm-vTgHMdBjUi1Qd5GiRIfW&index=5)
```

> And lastly, looking way back to September 2018, [Tobi's](https://twitter.com/TobiasBieniek/) talk from [EmberCamp Chicago](https://twitter.com/embercamp), "Internationalization: It's Easy in Ember" was released and is [now available on YouTube.](https://www.youtube.com/watch?v=K4nKWp1z4cU&t=0s&list=PL4eq2DPpyBbm-vTgHMdBjUi1Qd5GiRIfW&index=5)

Reviewer 3: I think it's fine to cover it here again if we didn't have it in any of the previous monthly updates. Most (or probably all) of what's in this post has been tweeted about already.

> And lastly, looking way back to September 2018, [Tobi's](https://twitter.com/TobiasBieniek/) talk from [EmberCamp Chicago](https://twitter.com/embercamp), "Internationalization: It's Easy in Ember" was released and is [now available on YouTube.](https://www.youtube.com/watch?v=K4nKWp1z4cU&t=0s&list=PL4eq2DPpyBbm-vTgHMdBjUi1Qd5GiRIfW&index=5)

Reviewer 4: yeah, fine by me. was just surprised to see it here 😉
